Scripts/SwiftLint.sh

```diff
@@ -26,8 +26,23 @@ if [ -z "$selected_path" ]; then
 	exit 1
 fi
 
+# temporary work directory
+scratch=`mktemp -d -t SwiftGen`
+function finish {
+  rm -rf "$scratch"
+}
+trap finish EXIT
+
+# actually run swiftlint
 if [ "$key" = "templates_generated" ]; then
-	find "${PROJECT_DIR}/${selected_path}" -name "*.swift" -exec ${PROJECT_DIR}/Scripts/swiftlint_no_disable_all.sh {} \;
+	# copy the generated output to a temp dir and strip the "swiftlint:disable:all"
+	for f in `find "${PROJECT_DIR}/${selected_path}" -name '*.swift'`; do
+		temp_file="${scratch}${f#"$PROJECT_DIR"}"
+		mkdir -p $(dirname "$temp_file")
+		sed s@"swiftlint:disable all"@" --"@ "$f" > "$temp_file"
```

Comment:
I wonder why I chose `@` instead of the usual `/` we commonly use there. Maybe the original script had file paths somewhere in there so I needed something other than `/`, but now that it's not really the case anymore, it's more common and thus easier to read to have `"s/swiftlint:disable all/ --/"` instead of `s@"swiftlint:disable all"@" --"@`

Reply:
I get this warning then:
> sed: 1: "s/swiftlint:disable": unterminated substitute pattern

Reply:
Woops, nvm, added quotes and it worked

Reply:
Even when you do wrap the whole thing in quotes?

Reply:
`sed "s/swiftlint:disable all/ --/" Tests/Fixtures/Generated/Strings/structured-swift4-context-multiple.swift` works from the Terminal for me at least